Ports/readline/package.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port=readline
+version=8.1
+useconfigure=true
+files="ftp://ftp.gnu.org/gnu/readline/readline-${version}.tar.gz readline-${version}.tar.gz"
```

Comment:
Sweet, I'll update the Python port to build with readline which should make the REPL experience a lot better :D